### ISSUE TITLE
`onPress` of `RadioButtonProps` is always ignored.

### ISSUE THREAD

**Issue by Haschikeks** [ThakurBallary/react-native-radio-buttons-group#73]
Hey,

I just noticed that, when you supply a `onPress` in the `RadioButtonProps`, it is always overwritten from the `RadioGroup`.
```typescript
        <RadioButton
          {...button}
          key={button.id}
          labelStyle={labelStyle}
          selected={button.id === selectedId}
          onPress={() => handlePress(button.id)}
        />
``` 
In a previous version the `onPress` looked like this 
``` 
onPress={(id: string) => {
  handlePress(id);
  if (button.onPress && typeof button.onPress === 'function') {
    button.onPress(id);
  }
}}
```
which I would prefer.

**Comment by ThakurBallary** [comment]
@Haschikeks thanks for writing. Fix is released in `v3.0.8`

**Comment by Haschikeks** [comment]
Hey @ThakurBallary 
thank you for the quick fix.
I also just noticed that the same "problem" is with the `selected` prop.
Although I'm not sure, this is as easily fixed.

For my project, I was able to use the `RadioGroup.selectedId`.
